addons/crm_project_issue/wizard/crm_lead2projectissue_wizard.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+
+class CrmLeadToProjectIssueWizard(models.TransientModel):
+    """ wizard to convert a Lead into a Project Issue and move the Mail Thread """
+    _name = "crm.lead2projectissue.wizard"
+    _inherit = 'crm.partner.binding'
+
+    project_id = fields.Many2one('project.project', string='Project', domain=[('use_issues', '=', True)])
+
+    @api.multi
+    def action_lead_to_project_issue(self):
+        # get the models
+        Issue = self.env['project.issue']
+        lead = self.env['crm.lead'].browse(self.env.context.get('active_id'))
+        for wizard in self:
+            partner_id = wizard._find_matching_partner()
+            if not partner_id and (lead.partner_name or lead.contact_name):
+                partner_id = lead.handle_partner_assignation()[lead.id]
+
+            # create new project.issue
+            vals = {
+                "name": lead.name,
+                "description": lead.description,
+                "email_from": lead.email_from,
+                "project_id": wizard.project_id.id,
+                "partner_id": partner_id,
+                "user_id": None
+            }
+            issue = Issue.create(vals)
```

Comment:
Hey @rpi-odoo you are spending unusually bytes behind variables so please keep pin it.
When you use once then assign directly. like : 
```
issue = self.env['project.issue'].create(vals)
```